mono/mini/mini.c

```diff
@@ -4210,7 +4210,7 @@ mono_jit_compile_method_inner (MonoMethod *method, MonoDomain *target_domain, in
 		break;
 	}
 	case MONO_EXCEPTION_MONO_ERROR:
-		// FIXME: MonoError has no copy ctor
+		// FIXME: ERROR_DECL (has) no copy ctor
```

Comment:
I think your regex ate a comment here. 😄

Reply:
Yes, this happened a few times and I fixed this one.